client/state/ui/guided-tours/contexts.js

```diff
@@ -7,65 +7,132 @@ import {
+ *
+ * @param {String} feature Name of feature
+ * @return {Function} Selector function
+ */
 export const isEnabled = feature => () =>
 	config.isEnabled( feature );
 
-export const isPreviewNotShowing = state =>
-	! isPreviewShowingSelector( state );
-
-export const isPreviewShowing = state =>
-	isPreviewShowingSelector( state );
-
+/*
+ * Returns milliseconds since user registration
+ *
+ * @param {Object} state Global state tree
+ * @return {Number|Boolean} Milliseconds since registration, false if cannot be determined
+ */
 const timeSinceUserRegistration = state => {
 	const user = getCurrentUser( state );
 	const registrationDate = user && Date.parse( user.date );
 	return registrationDate ? ( Date.now() - registrationDate ) : false;
 };
 
-const WEEK_IN_MILLISECONDS = 7 * 1000 * 3600 * 24;
+/*
+ * Returns true if user is considered "new" (less than week since registration)
+ *
+ * @param {Object} state Global state tree
+ * @return {Boolean} True if user is new, false otherwise
+ */
 export const isNewUser = state => {
 	const userAge = timeSinceUserRegistration( state );
 	return userAge !== false ? userAge <= WEEK_IN_MILLISECONDS : false;
 };
 
+/*
+ * Returns selector that tests if user is older than given time since registration
+ *
+ * @param {Number} age Number of milliseconds
+ * @return {Function} Selector function
+ */
 export const isUserOlderThan = age => state => {
 	const userAge = timeSinceUserRegistration( state );
 	return userAge !== false ? userAge >= age : false;
 };
 
+/*
+ * Returns selector that tests if user has registered before given date
+ *
+ * @param {Date} date Date of registration
+ * @return {Function} Selector function
+ */
 export const hasUserRegisteredBefore = date => state => {
 	const compareDate = date && Date.parse( date );
 	const user = getCurrentUser( state );
 	const registrationDate = user && Date.parse( user.date );
 	return ( registrationDate < compareDate );
 };
 
+/*
+ * Returns selector that tests whether user interacted with given component
+ *
+ * @param {String} componentName Name of component to test
+ * @return {Function} Selector function
+ */
 export const hasUserInteractedWithComponent = componentName => state =>
 	getLastAction( state ).component === componentName;
 
+/*
+ * Returns true if selected site can be previewed
+ *
+ * @param {Object} state Global state tree
+ * @return {Boolean} True if selected site can be previewed, false otherwise.
+ */
 export const isSelectedSitePreviewable = state =>
 	getSelectedSite( state ) && getSelectedSite( state ).is_previewable;
 
+/*
+ * Returns true if current user can run customizer for selected site
+ *
+ * @param {Object} state Global state tree
+ * @return {Boolean} True if user can run customizer, false otherwise.
+ */
 export const isSelectedSiteCustomizable = state =>
 	getSelectedSite( state ) && getSelectedSite( state ).is_customizable;
 
+/*
+ * Returns true if a/b test is in given variant
```

Comment:
There are a few inconsistencies wrt whether a new function or an actual result is returned. In a few places it says "Returns selector", and here e.g. it says "Returns true" (even though it returns a function / selector).